compiler/rustc_trait_selection/src/traits/fulfill.rs

```diff
@@ -631,7 +631,7 @@ impl<'a, 'tcx> ObligationProcessor for FulfillProcessor<'a, 'tcx> {
                 }
                 ty::PredicateKind::Clause(ty::ClauseKind::ConstArgHasType(ct, ty)) => {
                     match self.selcx.infcx.at(&obligation.cause, obligation.param_env).eq(
-                        DefineOpaqueTypes::No,
+                        DefineOpaqueTypes::Yes,
```

Comment:
This should never matter -- const types can't name TAITs (afaict), at least not without more features enabled. This can stay `Yes` but needs a FIXME.